nx/include/switch/services/sm.h

```diff
@@ -68,9 +133,14 @@ static inline void serviceClose(Service* s) {
     switch (s->type) {
 
     case ServiceType_Normal:
+    case ServiceType_Domain:
```

Comment:
What happens if you create a domain from an overriden service? It shouldn't use svcCloseHandle here.

Reply:
You shouldn't convert an overriden service to a domain to begin with...

Do overriden domains exist/are they desirable?

Reply:
FS comes to mind as a service that might be overriden; and that may benefit from domains iirc.

Reply:
Yeah we need to be able to override domains (but not subdomains)